dracut.sh

```diff
@@ -1446,7 +1446,14 @@ for line in "${fstab_lines[@]}"; do
                 push_host_devs "$i"
             done
         done
+    elif [[ $3 == zfs ]]; then
+        for mp in $1; do
+            for i in $(zfs_devs "$mp"); do
+                push_host_devs "$i"
+            done
```

Comment:
this isn't any different from `push_host_devs $(zfs_devs "$mp")`, is it?

Reply:
No, no it's not. 😅